enzy_htp/structure/residue.py

```diff
@@ -286,6 +286,14 @@ def is_solvent(self) -> bool:
         """Checks if the Residue() is an rd_sovlent as defined by enzy_htp.chemical.solvent.RD_SOLVENT"""
         return self._rtype == chem.ResidueType.SOLVENT
 
+    def is_counterions(self, counterion_list: List[str]=None) -> bool:
+        """Checks if the Residue() is 
+        counter ion. default only Na+ and Cl- counters.
+        This list can be overwritten by {counterion_list}"""
+        if not counterion_list:
```

Comment:
probably change to `if counterion_list is None` since this logic technically works for an empty list also

Reply:
Make sense. Made the change